libs/accounts/rate-limit/src/lib/rate-limit.spec.ts

```diff
@@ -98,6 +98,57 @@ describe('rate-limit', () => {
     );
   });
 
+  it('can ignore certain emails', () => {
+    rateLimit = new RateLimit(
+      {
+        rules: {},
+        ignoreEmails: ['@mozilla.com$', 'foo@firefox.com'],
```

Comment:
I always feel a bit nervous using regex in these types of filters. Mostly because of the chance to mess them up and allow unintended things. Should we do this based on specific domains or explict emails?

Reply:
I think this is how it worked previously. I do understand this concern though... There are some cases where it makes sense to filter by domain, and not explicitly by email address. Let me file a polish PR where we can do this by ignoreEmails and ignoreEmailDomains.